test_apps/smokeTests/smokeTests.py

```diff
@@ -0,0 +1,237 @@
+#! /usr/local/bin/python3
+#! /usr/bin/python3
```

Comment:
Can you have two shebangs in a script? Why not simply use the 'env' command for portability:

#!/usr/bin/env python3 

should work just about everywhere

Reply:
Thanks - Did not know that.